.github/workflows/test.yml

```
@@ -22,7 +22,7 @@ jobs:
             rs: 1.85.0
           - os: macos-latest
             rs: 1.86.0
-        features: ["", "--features unstable,legacy,__abi-generate"]
+        features: ["", "--features unstable,legacy,__abi-generate,deterministic-account-ids"]
```

How about running doctests in CI as well?
```suggestion
        features: ["", "--features unstable,legacy,__abi-generate,deterministic-account-ids"]
        targets: ["--all-targets", "--doc"]
```

And then add it to the step below:
```yaml
- name: test
   run: cargo test --workspace ${{ matrix.targets }} ${{ matrix.features }}
```